### ISSUE TITLE
Issue - often deleting the wrong image

### ISSUE THREAD

**Issue by iwejfnd** [AlUlkesh/sd_delete_button#5]
I've been using this for a couple days now and have noticed that when I have a certain image selected in the txt2img page and delete it, it often deletes one of the other images that I have generated in a batch.

**Comment by AlUlkesh** [comment]
Yes, it always deletes the last generated image (skipping grids).

I agree that that is not exactly intuitive in a batch situation. Obviously it would make more sense there to delete the selected image.

However, what would you expect to happen, when the generation is just finished, no image is selected yet and delete is pressed?

**Comment by Targren** [comment]
> However, what would you expect to happen, when the generation is just finished, no image is selected yet and delete is pressed?

Ideally, nothing.  If the behavior is ambiguous or unintuitive, and the operation is destructive, better to default to not doing anything if there's more than 1 image generated and no valid image is selected yet. 

Just  my $0.02 though.
